tests/App/Admin/AuthorAdmin.php

```diff
@@ -25,7 +29,17 @@ protected function configureListFields(ListMapper $list): void
     {
         $list
             ->add('id')
-            ->addIdentifier('name');
+            ->addIdentifier('name')
+            ->add('number_of_books', FieldDescriptionInterface::TYPE_INTEGER, [
+                'accessor' => static function (Author $author): int {
+                    return $author->getBooks()->count();
+                },
+                'template' => 'author/list_number_of_books_field.html.twig',
+            ])
+            ->add('numberOfReaders', FieldDescriptionInterface::TYPE_INTEGER, [
+                'template' => 'author/list_number_of_readers_field.html.twig',
+            ])
+            ;
```

Comment:
IIRC, this should be caught by PHP-CS-Fixer.
```suggestion
            ]);
```

Reply:
Do you want to make a PR on dev-kit for the `multiline_whitespace_before_semicolons` rule ?

Reply:
I thought that it was already added. I'll provide a PR later.